lib/factories/ppt/slides.js

```diff
@@ -111,10 +110,21 @@ class SlideFactory {
         let slideKey = `ppt/slides/${slide.name}.xml`;
         let objectCount = 0;
 
-        let imageX = options.x ? PptxUnitHelper.fromPixels(options.x) : 0;
-        let imageY = options.y ? PptxUnitHelper.fromPixels(options.y) : 591502;
-        let imageCx = options.cx ? PptxUnitHelper.fromPixels(options.cx) : 9144000;
-        let imageCy = options.cy ? PptxUnitHelper.fromPixels(options.cy) : 5674995;
+        options = Object.assign(
+            {
+                // defaults
+                x: 100,
```

Comment:
These values changed because they needed to have `fromPixels`  run on them now?

Reply:
Yeah.